src/sagemaker/estimator.py

```diff
@@ -1007,6 +1007,7 @@ def register(
         if compile_model_family is not None:
             model = self._compiled_models[compile_model_family]
         else:
+            kwargs["model_kms_key"] = self.output_kms_key
```

Comment:
what if `model_kms_key` is already in kwargs?

Reply:
good catch, I will fix it